docs/apache-airflow-providers-microsoft-azure/operators/azure_synapse.rst

```diff
@@ -41,6 +41,17 @@ Below is an example of using this operator to execute a Spark application on Azu
       :end-before: [END howto_operator_azure_synapse]
 
 
+AzureSynapseRunPipelineOperator
+-----------------------------------
+Use the :class:`~airflow.providers.microsoft.azure.operators.synapse.AzureSynapseRunPipelineOperator` to execute a pipeline application within Synapse Analytics.
+By default, the operator will periodically check on the status of the executed pipeline to terminate with a "Succeeded" status.
+
+.. exampleinclude:: /../../tests/system/providers/microsoft/azure/example_azure_synapse.py
+      :language: python
+      :dedent: 4
+      :start-after: [START howto_operator_azure_synapse]
+      :end-before: [END howto_operator_azure_synapse]
```

Comment:
This reference already exist for AzureSynapseRunSparkBatchOperator.
If we don't have example for `AzureSynapseRunPipelineOperator` then you need to create one give it a new reference and then you can utalize it here.

For example:

https://github.com/apache/airflow/blob/63cc915cd38a5034df6bf9c618e12f8690eeade0/tests/system/providers/microsoft/azure/example_azure_synapse.py#L64-L70

Reply:
![Airflow code snippet](https://github.com/apache/airflow/assets/109049037/530cdd0d-2526-4304-9d5f-c262c94d651a)

I saw this @eladkal for `AzureSynapseRunPipelineOperator`. Can I reference this?

Reply:
Yes. you will need to add `[START howto_operator_azure_synapse_run_pipeline]`  /  `[END howto_operator_azure_synapse_run_pipeline]` then you can reference it. Check the link I shared above